tests/instancetype/instancetype.go

```diff
@@ -41,7 +41,10 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-const timeout = 300 * time.Second
+const (
+	timeout                    = 300 * time.Second
+	PreferenceMemoryBelowGuest = 64 * 1024 * 1024
```

Comment:
```suggestion
	PreferenceMemoryBelowGuest = 64 * 1024 * 1024 // 64 KiB
```

Reply:
Silly suggestion, fell free to ignore it

Reply:
If that is only within a single test case I'd move that const there.